src/main/java/com/sap/cloudfoundry/client/facade/rest/CloudControllerRestClientImpl.java

```diff
@@ -2079,7 +2095,7 @@ private void doDeleteDomain(UUID guid) {
     }
 
     private void doDeleteRoute(UUID guid) {
-        delegate.routes()
+        delegate.routesV3()
```

Comment:
Deletion of routes return job id which we have to check is it really deleted or we have to poll deletion.